rasa/core/migrate.py

```diff
@@ -245,6 +245,7 @@ def migrate_domain_format(
     if domain_file.is_dir():
         backup_dir = current_dir / "original_domain"
         backup_dir.mkdir()
+        backup_path = backup_dir / "original_domain.yml"
```

Comment:
Oh this line is not necessary, you can rename `backup_dir` to `backup_path`, that's what i meant.

Reply:
Okay - so the message will contain either a path to a directory, or a path to a file.

Reply:
Yup, exactly!